core/templates/dev/head/pages/collection_player/collection_player.html

```diff
@@ -141,11 +136,7 @@ <h2 ng-if="collectionPlaythrough.hasFinishedCollection()" class="oppia-page-head
                                   is-community-owned="summaryToPreview.community_owned"
                                   style="position: absolute; left: 75px; z-index: 10;"/>
       </div>
-    </div>
-      {% else %}
-        <div class="oppia-collection-player-small-text">You must login to save your progress.</div>
-        <collection-node-list collection-id="collectionId" collection-nodes="collection.getCollectionNodes()"></collection-node-list>
-      {% endif %}
+      </div>
```

Comment:
Something wrong with indentation here? (It's on the same level as the previous line...)